### ISSUE TITLE
Emit node new data in drop event

### ISSUE THREAD

**Issue by MWStudio4** [phphe/vue-draggable-nested-tree#20]
Hello. Try to use you component. In my project case I need to know the new node place data to prevent place node with children to lower level (so need to know the new level). But i can't get it in DROP hook or CHANGE event. I think it would be nice to emit 'dplh' too

**Comment by phphe** [comment]
yes. the node level will not update in event 'change' immediately.
but its parent is updated then. you can get its level like this: node.parent.level + 1
you can get dplh like this: anyTree.dplh

**Comment by MWStudio4** [comment]
Yes. It works. Thanks
